### ISSUE TITLE
an example to get DLL from dump

### ISSUE THREAD

**Issue by ttodua** [fremag/MemoScope.Net#212]
We can view DLL objects and every property of them... but is there any way, and if so, an example - how to get dll to write to a normal .dll file for further usage for JustDecompile ( or like that)?

**Comment by fremag** [comment]
The dll is not in the memory dump: it contains only data, no executable code so I don't think it's possible.

**Comment by Muppetdog** [comment]
If its a full memory dump, the full dlls are in the dump.  I pull them out and reflect them all the time.

**Comment by TheXenocide** [comment]
Right, this very much depends on whether the dump is a full dump or a minidump

**Comment by ttodua** [comment]
yes, full dump 

@Muppetdog can you point me (where can i read) how to do, step by step?

**Comment by fremag** [comment]
I'm sorryb ut I've no idea how to do that.

Memoscope is just a UI: all features come from ClrMd and I can't find anything in this to extract dlls.
I know ClrMd 2.0 is coming soon, I'll check then.

**Comment by Muppetdog** [comment]
Easiest way to dump out dolls from a dump is with the Mex debugging extension for Windbg.

!writemodule -a -p pathtosave  

It will save all the modules in the dump, managed and unmanaged

https://www.microsoft.com/en-us/download/details.aspx?id=53304

**Comment by ttodua** [comment]
@fremag , ah, sorry, seems i have made [duplicate  ](https://github.com/fremag/MemoScope.Net/issues/215#issuecomment-683335157 ) request for that, however, what your thoughts about last comment by @Muppetdog ?  Could that be integrated?
